test/functional/FunctionalBadProjectSpec.hs

```diff
@@ -32,7 +32,7 @@ spec = describe "behaviour on malformed projects" $ do
           d ^. code `shouldBe` Nothing
           d ^. source `shouldBe` Just "ghcmod"
           d ^. message `shouldBe`
-            (T.pack "readCreateProcess: stack \"build\" \"--only-configure\" \".\" (exit 1): failed\n")
+            (T.pack "callProcessStderr: stack --stack-yaml=/root/build/test/testdata/badProjects/cabal/stack.yaml build --only-configure . (exit 1): failed\n")
```

Comment:
For the HaRe tests I copy certain projects to $TEMPDIR/testproj, and then work with that.  This looks like a very brittle hard coded path.

Reply:
I never even noticed the path, I just copied and pasted it verbatim. Will do a fuzzier expectation